<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AvatarsVSRooks</class>
 <widget class="QMainWindow" name="AvatarsVSRooks">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>232</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Informacin">
       <property name="text">
        <string>Ingrese su nombre de usuario y Contraseña</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtUsuario">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Usuario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPassword">
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Contraseña</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEntrar">
       <property name="text">
        <string>Entrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
